flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/LocalExecutor.java

```diff
@@ -232,7 +238,10 @@ public void useCatalog(SessionContext session, String catalogName) throws SqlExe
 			.getTableEnvironment();
```

Comment:
@bowenli86 please remove `useCatalog` and `useDatabase` from the executor interface. The functionality is already covered by `org.apache.flink.table.client.gateway.local.LocalExecutor#validateSession`.

Reply:
@twalthr can you elaborate a bit more? AFAICT, `org.apache.flink.table.client.gateway.local.LocalExecutor#validateSession` seems only to be responsible for creating an execution context, but cannot set any current catalog and current database?

Reply:
`useCatalog` and `useDatabase` are APIs for CliClient when running SQL statements such as "USE CATALOG XXX". Are we talking about the same thing?